src/material-examples/chips-drag-and-drop/chips-drag-drop-example.css

```diff
@@ -0,0 +1,7 @@
+.cdk-drag-animating {
```

Comment:
@crisbeto can we gate this on another CSS class as well? I forget what you have the ability to add

Reply:
@jelbourn I have wrapped this definition within another class. Almost the same as @crisbeto proposal. Is it fine for you now?